src/RollImage.cpp

```diff
@@ -4340,13 +4328,15 @@ void RollImage::generateHoleMidifile(MidiFile& midifile) {
 void RollImage::setMidiFileTempo(MidiFile& midifile) {
 	if (m_rollType == "welte-red") {
 		// TPQ is 6 times the tempo at 300 DPI, (so 591 = 6 * 98.5)
-		// 591 is for Red Welte tempo of 98.5 (~3 meters/minute).
-		midifile.setTPQ(6 * 98.5);
+		// 591 is for Red Welte tempo of 98.5 (~3 meters/minute)
+		// 568 is the currently preferred value, however.
+		midifile.setTPQ(568);
 	} else if (m_rollType == "welte-green") {
 		// Peter Phillips's dissertation describes the speed for green Welte
 		// rolls as "seven feet per minute" (p. 121). 7ft/min = 2.1336m/min =
-		// tempo of 70.0532 (* 6 = 420)
-		midifile.setTPQ(6 * 70);
+		// tempo of 70.0532 (* 6 = 420).
+		midifile.setTPQ(420);
+		setRollAcceleration(.22336);
```

Comment:
Would it be better to set the roll acceleration explicitly here for all the roll types?  Right now it seems that the default value for Welte Red is falling through for everything except the Welte Green -- rather than setting the default in another file (and having the explanation there, rather than here), could it be done more explicitly here?

Reply:
Okay, cool, thanks -- this is an improvement that goes a long way to addressing my point.  This is code that will be modified again in the future, for sure, and there will be further opportunities to improve the situation then, I suppose.  Thanks!